### DISCUSSION TITLE
For icons, the character width is inconsistent in different terminals.

### DISCUSSION THREAD

**Discussion by XunHang** [LazyVim/LazyVim#1659] [Q&A] [answered]
I am using lazyvim 0.9.8 with neovim 0.9.4 on ubuntu 20.04, and I discovered a strange thing today.
In gnome terminal, icons will be rendered beautiful, but in kitty and alacritty (both of them are based on GPU), icon will be smaller, here is the compare image:
![image](https://github.com/LazyVim/LazyVim/assets/17587962/a9845875-c72a-42a2-a0f8-868662deb693)

Kitty terminal's doc tell about this: https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.narrow_symbols, but it does not work on my machine.

It also looks like https://github.com/LazyVim/LazyVim/issues/1571, I know very little about font rendering, so I don’t know if it’s related to it.

Kitty terminal is only one of Lazyvim recommended terminals that supports ligatures on the Linux platform, so I hope to find the cause of the problem.

Does anyone know?

**Comment by folke** [comment, marked as answer]
For kitty, you can add this:

```bash
symbol_map U+23FB-U+23FE,U+2665,U+26A1,U+2B58,U+E000-U+E00A,U+E0A0-U+E0A3,U+E0B0-U+E0C8,U+E0CA,U+E0CC-U+E0D2,U+E0D4,U+E200-U+E2A9,U+E300-U+E3E3,U+E5FA-U+E634,U+E700-U+E7C5,U+EA60-U+EBEB,U+F000-U+F2E0,U+F300-U+F32F,U+F400-U+F4A9,U+F500-U+F8FF Symbols Nerd Font Mono
```

My kitty:

![image](https://github.com/LazyVim/LazyVim/assets/292349/262e2ef5-d03f-4ded-b572-9325bf3ad27e)

**Reply by XunHang** [reply]
Thanks! But things get a little weird...

![image](https://github.com/LazyVim/LazyVim/assets/17587962/2c9fc38e-700e-4bdf-ba7d-0b9c8d2b8380)

Maybe I should try to use you font?

**Reply by dpetka2001** [reply]
You should install [nerd symbols](https://github.com/ryanoasis/nerd-fonts/releases/download/v3.0.2/NerdFontsSymbolsOnly.zip) if you want to use that in kitty.

**Reply by XunHang** [reply]
Wooooooow!!! It works! Thank you!
